portfolio/src/main/webapp/style.css

```diff
@@ -14,10 +14,69 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.header {
+  overflow: hidden;
+  background-color: #f1f1f1;
+  padding: 20px 10px;
+}
+
+
+/* Style the header links */
+.header a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 12px;
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 25px;
+  border-radius: 4px;
+}
+
+/* Change the background color on mouse-over */
+.header a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Style the active/current link*/
+.header a.active {
+  background-color: dodgerblue;
+  color: white;
+}
+
+/* Float the link section to the right */
+.header-right {
+  float: right;
+}
+
+ul{
+    display: flex;
+  flex-wrap: wrap;
+  /* padding: 0 4px; */
```

Comment:
nit: Looks like leftovers from debugging?
Same for the li element below

Reply:
Yea I think I didn't clean it up very well -- I'll have it fixed!